spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -99,5 +99,14 @@
         )
       end
     end
+
+    context 'When the user is VIP' do
+      let(:user) { create(:user, vip: true) }
+      let!(:user_targets) { create_list(:target, 5, user:) }
+
+      it 'creates unlimited targets' do
+        expect { subject }.to change { Target.count }
```

Comment:
what about checking this count is incremented by 5 or from 0 to 5 ?

Reply:
I modified the spec, but with the comment that Seba made, I adjusted the count to increase by 1 so that the maximum target would be more than 3 if VIP.